server/run-tests.sh

```diff
@@ -13,41 +11,52 @@ export SVIX_DB_POOL_MAX_SIZE="500"
 export SVIX_REDIS_POOL_MAX_SIZE="10000"
 
 echo "*********** RUN 1 ***********"
-SVIX_QUEUE_TYPE="redis" \
-SVIX_CACHE_TYPE="redis" \
-SVIX_REDIS_DSN="redis://localhost:6379" \
-${TEST_COMMAND}
+(
+    export SVIX_QUEUE_TYPE="redis"
+    export SVIX_CACHE_TYPE="redis"
+    export SVIX_REDIS_DSN="redis://localhost:6379"
+    cargo test
+    cargo test -- --ignored redis
```

Comment:
Chiming in re: my gripe about possible different features being selected. Looks like this was fixed by not having a `TEST_COMMAND` as for the general case, which I think is fair.

This way, it should be clear if a given run needs to select features, it should happen for both `cargo test` invocations 👍🏻 

It's less DRY, but tests and related should not be DRY at the expense of information splitting.

The reliance on `#[ignore]` still feels a little fiddly to me, but I think this is better than it was when I last looked.